test/models/vendor_assignment_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-class VendorAssigmentTest < ActiveSupport::TestCase
+class VendorAssignmentTest < ActiveSupport::TestCase
```

Comment:
Yeah, there are a lot of empty tests, it might be better just to delete them all (possibly along with all those fixtures).

I had to fix up a whole load of fixtures just to get the three tests I wrote to run, I don't think the main test suite will run quite yet. But for now I'm just taking a very light approach to getting this stuff to work. Like you I'm very new to the codebase, so don't see the value in going full steam ahead right now.

I think a chat about how we approach testing rather soon would be beneficial.